pkg/moov/transfer_models.go

```diff
@@ -126,6 +128,26 @@ type Transfer struct {
 	SalesTaxAmount *Amount `json:"salesTaxAmount,omitempty"`
 }
 
+// Fees charged to accounts involved in the transfer.
+type MoovFees struct {
+	// ID of the account that fees were charged to.
+	AccountID string `json:"accountID"`
+	// List of fee IDs that sum to the totalAmount.
+	FeeIDs []string `json:"feeIDs"`
+	// The total amount of fees charged to the account.
+	AmountDecimal AmountDecimal `json:"totalAmount"`
```

Comment:
```suggestion
	TotalAmount AmountDecimal `json:"totalAmount"`
```
We can leave the field name as before -- other suggestion was just on the type name